client/src/pages/profile/index.tsx

```diff
@@ -0,0 +1,170 @@
+import ApplicationLayout from 'layouts/application';
+import Head from 'next/head';
+import { Label, Input } from 'components/forms';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { useForm } from 'react-hook-form';
+import { useMutation } from 'react-query';
+import { useCallback } from 'react';
+import { apiService } from 'services/api';
```

Comment:
sort the dependencies
```suggestion
import { useCallback } from 'react';
import Head from 'next/head';
import * as yup from 'yup';
import { yupResolver } from '@hookform/resolvers/yup';
import { useForm } from 'react-hook-form';
import { useMutation } from 'react-query';

import ApplicationLayout from 'layouts/application';
import { apiService } from 'services/api';
import { Label, Input } from 'components/forms';
```